src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,11 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList: FC<{ todos:Array<Todo> }> = ({ todos }) => (
```

Comment:
create interface TodoListProps
```suggestion
export const TodoList: FC<TodoListProps> = ({ todos }) => (
```